docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/pam/master-docinfo.xml

```diff
@@ -0,0 +1,10 @@
+<productname>{PRODUCT}</productname>
+<productnumber>{PRODUCT_VERSION}</productnumber>
+<subtitle>
+	For {PRODUCT} {PRODUCT_VERSION}
+
+</subtitle>
+<abstract>
+	<para>This document describes how to build custom dashboard widgets in {PRODUCT} {PRODUCT_VERSION}.</para>
```

Comment:
"custom dashboard widgets" => custom pages including dashboards and reports.

Reply:
sentence changed as suggested.